astropath/bayesian.py

```diff
@@ -0,0 +1,190 @@
+"""Methods related to Bayesian association analysis"""
+
+import numpy as np
+
+import copy
+
+from astropy import units
+from astropy.coordinates import SkyCoord
+
+from IPython import embed
+
+
+def raw_prior_Oi(Pchance, Sigma_m, method, half_light=None):
+    """
+    Raw prior for a given set of Pchance values and/or n(m)
+
+    Args:
+        Pchance (float or np.ndarray):
+            Chance probability
+        Sigma_m (float or np.ndarray):
+            Number density of sources on the sky brighter than m
+        method (str):
+            inverse :: Assign inverse to Sigma_m
+            inverse1 :: Assign inverse to Sigma_m * half_light
+            inverse2 :: Assign inverse to Sigma_m * half_light**2
+            orig_inverse :: Assign inverse to P_chance
+            identical :: All the same
+            linear :: 1-Pchance (not recommended)
+        half_light (float or np.ndarray, optional):
```

Comment:
It would be useful to mention the units for Sigma_m and half_light in these docstrings.

Reply:
done